### DISCUSSION TITLE
How to display a snackbar on Windows ?

### DISCUSSION THREAD

**Discussion by BeepBeepBopBop** [CommunityToolkit/Maui#2020] [Q&A] [answered]
Hello, I am trying to display a snackbar alert on Windows without success.

Actually when clicking "display snackbar" buttons in the sample application, nothing happens even though it does not say anywhere that Windows is not supported.  Is there anything I am missing ?

Thanks for your help

**Comment by TheCodeTraveler** [comment, marked as answer]
[Enable notifications](https://support.microsoft.com/en-us/windows/change-notification-and-quick-settings-in-windows-ddcbbcd4-0a02-f6e4-fe14-6766d850f294) on Windows

**Reply by BeepBeepBopBop** [reply]
Thanks for the quick reply, they were indeed turned off. I guess snackbar/toasts should not be used on Windows to display important information such as errors then.

**Comment by CliffAgius** [comment]
Closed as Answered.
